README.md

```diff
@@ -1,6 +1,7 @@
 ![CI status](https://github.com/seattle-uat/civiform/actions/workflows/push.yaml/badge.svg)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/6008/badge)](https://bestpractices.coreinfrastructure.org/projects/6008)
 [![Seattle Staging Deploy](https://github.com/seattle-uat/civiform-deploy/actions/workflows/deploy-staging.yml/badge.svg?branch=main)](https://github.com/seattle-uat/civiform-deploy/actions/workflows/deploy-staging.yml)
+[![coverage](https://codecov.io/gh/seattle-uat/civiform/branch/main/graph/badge.svg?token=K7RXZGHECF)](https://codecov.io/gh/seattle-uat/civiform)
```

Comment:
What is the token for? Is it a secret? Can we avoid adding the token?

Reply:
Agreed! not needed.. found another way.

Reply:
Blocking: Inlining bion@'s comment:

> If it's a secret it needs to be retired and replaced since it's now leaked
into git history

Assuming it wasn't a secret, then this comment is not blocking.

Reply:
Regarding your comment on the leak of the Token - this token is not the uploading token (which is a different and needs to be kept secret) this is just the token to access the badge. Also Since we are a public Repo - use of token is not needed for us. I am trying to find more info on how to reset the badge token but so far, it looks like nothing important is lost. I will keep digging.